packages/core/src/index.js

```diff
@@ -35,6 +35,7 @@ import ListRow from './ListRow';
 import ColumnView from './ColumnView';
 import Overlay from './Overlay';
 import Popup from './Popup';
+import PopupButton from './PopupButton';
```

Comment:
Forget to export `<PopupIcon>` and `<PopupMessage>` too?
In not, they shouldn't be exportable.

Reply:
But they need to be imported for tests?

Reply:
Oh ya, I forget tests 😛

Reply:
When I was doing this, I thought they're just helper components to be used inside <Popup> that transform **strings** into elements with default settings. You don't usually need to pass elements to `icon` and `message` props, unlike the `buttons` prop which expects an array of elements.

That's why I felt it's better to extract `<PopupButton>` out as an independent component (and thus export it to the main bundle), as it's expected to be used a lot. Meanwhile, `<PopupIcon>` and `<PopupMessage>` are not.

I'm open to export the later two components though. What do you say?

Reply:
I think it is okay to keep them as the internal components, since they are only in popup. 👍